### ISSUE TITLE
Crypto.Util.number.size is a bit slow

### ISSUE THREAD

**Issue by jneb** [Legrandin/pycryptodome#479]
I saw that the code for Crypto.Util.number.size is:
```
def size (N):
    """Returns the size of the number N in bits."""
    bits = 0
    while N >> bits:
        bits += 1
    return bits

```
How about:
`return N.bit_length()`
?

**Comment by Legrandin** [comment]
Good point. This `size` function is a left-over from Python 2.6 and older, which are not supported anymore.

**Comment by jneb** [comment]
Pull request #484 addresses this issue and does two more improvements.

**Comment by Legrandin** [comment]
Fixed with a31d1634b0591570fa744559c62232fdce693eec
